project/plugins.sbt

```diff
@@ -4,3 +4,7 @@ addSbtPlugin("me.lessis" % "bintray-sbt" % "0.2.1")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
+resolvers += "Sonatype OSS Releases" at "https://oss.sonatype.org/service/local/staging/deploy/maven2"
```

Comment:
Is this necessary? I think the plugin is published on Maven central: http://repo1.maven.org/maven2/org/scalariform/sbt-scalariform_2.10_0.13/1.6.0/